odk/odk.py

```diff
@@ -507,7 +507,13 @@ class OntologyProject(JsonSchemaMixin):
     
     use_mappings : bool = False
     """if true use SSSOM mapping files."""
+
+    set_docker_user : bool = False
+    """if true continue to use your own user when using docker, instead of the default docker user."""
```

Comment:
Making that configurable at the ODK level is a bad idea IMO. This bakes the setting into the generated `run.sh` file, and so assumes all editors of the ontology need the same setting (either they all need to run Docker with their own privileges or none of them does). This will not always be the case, if for example one editor is working under macOS and another is working on GNU/Linux.

It can even happen that the _same_ editor is sometimes using macOS and sometimes using GNU/Linux: I do most of my ontology editing work on macOS, but for FBcv, I have to do the release on GNU/Linux, because my macOS machine does not have enough memory.

If we make that configurable at the ODK level, it would be better to invert the default: the default should be to run with the user’s credentials, and only if an ontology has a workflow that requires the ability to install supplementary packages should that setting be changed.

Reply:
I would agree with this but it introduces a breaking change. It literally took me hours, or days, to figure out why my `oak` pipelines stopped working (oak is trying to cache data in the user directory `~/.data`, which does not exist when you go in with `-u`). Literally half (or more) of my pipelines have some kind of requirements.txt step in them - and they just fail with obscure errors. We should avoid introducing breaking changes as much as possible..

Reply:
Solution on its way in #768.

Reply:
We dropped this from 1.4 and I moved your PR to 1.5